specificatie/domain.yaml

```diff
@@ -203,6 +203,15 @@ components:
                         De identificaties van de stukdelen (paragrafen in een akte met een rechtsfeit) waarin dit beslag is vermeld
           items:
             type: "string"
+    BijbehorendAppartementsrecht:
```

Comment:
Wat Cathy bedoel en wat ik eerder heb aangegeven is of dit niet gewoon Appartementsrecht moet heten